src/js/components/award/contract/ContractDetails.jsx

```diff
@@ -37,9 +40,9 @@ export default class ContractDetails extends React.Component {
                         }} />
                     <table>
                         <tbody>
-                            <DetailRow
+                            <AwardDescription
+                                selectedAward={award}
                                 title="Description"
-                                value={award.description}
                                 overflow={award.description.length > maxChars} />
```

Comment:
The overflow prop should be calculated within the award description container using the length of the new description value.